spec/support/acceptance_spec_helpers.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module AcceptanceSpecHelpers
+  def generate_model(model_name, *fields)
+    Rails::Generators.invoke('declare_schema:model', [model_name, *fields])
+  end
+
+  def generate_migrations(*flags)
+    Rails::Generators.invoke('declare_schema:migration', flags)
+  end
+
+  def expect_model_definition_to_eq(model, expectation)
+    model_file_path = "#{TESTAPP_PATH}/app/models/#{model}.rb"
+
+    expect(File.exist?(model_file_path)).to be_truthy
+    expect(File.read(model_file_path)).to eq(expectation)
+  end
+
+  def expect_test_definition_to_eq(model, expectation)
+    test_file_path = "#{TESTAPP_PATH}/test/models/#{model}_test.rb"
+
+    expect(File.exist?(test_file_path)).to be_truthy
+    expect(File.read(test_file_path)).to eq(expectation)
+  end
+
+  def expect_test_fixture_to_eq(model, expectation)
+    fixture_file_path = "#{TESTAPP_PATH}/test/fixtures/#{model}.yml"
+
+    expect(File.exist?(fixture_file_path)).to be_truthy
+    expect(File.read(fixture_file_path)).to eq(expectation)
+  end
```

Comment:
These expectations look the same
```
    expect(File.exist?(model_file_path)).to be_truthy
    expect(File.read(model_file_path)).to eq(expectation)
```
Any argument against factoring that part out into a different method?

Reply:
Absolutely. Went ahead and dried this up.